packages/salesforcedx-vscode-core/test/vscode-integration/commands/forceOrgOpen.test.ts

```diff
@@ -38,17 +38,13 @@ describe('Force Org Open', () => {
   });
 
   describe('Executor is chosen based on environment', () => {
-    let originalEnvValue: string;
-
-    beforeEach(() => {
-      originalEnvValue = process.env.SFDX_CONTAINER_MODE as string;
```

Comment:
Jonny, i found that there was some side-effects from us trying to restore the original value.  It took me a while to figure it out, but by the time my tests ran process.env.SFDX_CONTAINER_MODE became the string 'undefined' instead of the value undefined.  I don't know exactly how or why, but yeah, that happened...  So now we're just going to delete it so there are definitely no side-effects.